src/nemos/basis.py

```diff
@@ -771,34 +769,73 @@ def evaluate_on_grid(self, n_samples: int) -> Tuple[NDArray, NDArray]:
         return super().evaluate_on_grid(n_samples)
 
 
-class RaisedCosineBasis(Basis, abc.ABC):
-    def __init__(self, n_basis_funcs: int) -> None:
+class RaisedCosineBasisLinear(Basis):
+    """Represent linearly-spaced raised cosine basis functions.
+
+    This implementation is based on the cosine bumps used by Pillow et al.[$^1$](references)
+    to uniformly tile the internal points of the domain.
```

Comment:
This phrase is confusing, should be explained somewhere.

Reply:
I'll add an example in the plot 1D basis function.  I think we should have a pr in which we work out a better description of the basis function and their properties. I started one in which i tried to make a table of the various properties.

Reply:
I think we should have  a minimal example here

Reply:
I added the issue #72 that would be aimed to improve those docstrings